<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>400</width><height>300</height></rect>
  </property>
  <property name="windowTitle"><string>8-Puzzle Solver</string></property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="algorithmComboBox">
    <property name="geometry"><rect><x>50</x><y>30</y><width>300</width><height>32</height></rect></property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry"><rect><x>50</x><y>80</y><width>300</width><height>40</height></rect></property>
    <property name="text"><string>Bắt đầu giải</string></property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry"><rect><x>50</x><y>140</y><width>300</width><height>100</height></rect></property>
    <property name="text"><string>Chi phí và bước sẽ hiển thị ở đây.</string></property>
    <property name="alignment"><set>Qt::AlignCenter</set></property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
